src/bundle/Resources/public/scss/ckeditor/_general.scss

```diff
@@ -0,0 +1,127 @@
+.ez-data-source__richtext {
+    background-color: $ez-white;
+    min-height: calculateRem(100px);
+    border: calculateRem(1px) solid rgba(0, 0, 0, 0.15);
+    border-radius: calculateRem(4px);
```

Comment:
`$ibexa-border-radius`?

Reply:
richtext should not rely on admin-ui variables